notepad.py

```diff
@@ -165,6 +172,27 @@ def onMessage(message, client):
 print('Joined room')
 room.send_message('[notepad] Hi o/')
 
+# Load timers
+try:
+    with open(timersFilename, 'rb') as f:
+        timersToLoad = pickle.load(f)
+
+        for item in timersToLoad:
+            diff = item['time'] - datetime.utcnow()
+
+            # Filter out expired timers
+            if diff < timedelta(0):
+                continue
+            
+            timers.append(item)
+
+            msg = client.get_message(item['messageId'])
+
+            t = Timer(diff.total_seconds(), reminder, args=(msg,))
+            t.start()
+except:
```

Comment:
Don't ever write all-in `except` clauses. Think of the kinds of exceptions that you anticipate and handle those, so that unexpected errors can bubble up to the user. If there are only single steps that may fail you can move the try-except inside the `with` block, for instance to check if the `pickle.load` failed and give the user an informative warning (as well as empty-initializing the timer list).

Reply:
I think one last catch-all in the routine that handles the message makes sense, so the problem is caught there and reported to chat instead of the thing just crashing. Would you agree?

Reply:
@BaumMitAugen See my latest commit below "Better error handling.". The idea is when it restarts to build the list of timers/initialize them. If there is an error, it would be better to see them in the console, as they don't really effect the running of the bot (It will skip initialization of the timers, and continue along with running of the bot). We can move that to sending a message to the chatroom if you prefer.